pages/k8s/upgrade-notes.md

```diff
@@ -196,7 +196,7 @@ the Docker subordinate includes clean-up code to uninstall Docker when the
 Docker charm is replaced with the containerd charm.
 
 ```bash
-juju deploy cs:~containers/docker
+juju deploy docker
```

Comment:
does this work? I thought we'd dropped the charm so I don't know why it would be in ch at all

Reply:
Okay, I see that it exists. This makes me more concerned. It isn't really a docs issue, but why are we pushing versions of theis charm to CH, especially the 1.24/stable channel?

Reply:
to be more verbose, the inclusion of a 1.24 version of this makes it look like it is supported in some way

Reply:
can we remove a channel. again, this is noting to do with docs really, and we can accept this change, but I think we should only have a latest/stable or whatever of docker, and maybe update its docs to indicate that it has ceased to be.